ember-cli/2017-11/november-02.md

```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-11-02
+
+## Attendees
+
+
+## Agenda
```

Comment:
Please reply here with agenda items

Reply:
* Concurrency issues
  * https://github.com/ember-cli/ember-cli/issues/7379
  * https://github.com/ember-cli/ember-cli/issues/7392
* `.ember-cli` as JS? https://github.com/ember-cli/ember-cli/issues/7336
* Update `sane` to latest https://github.com/ember-cli/ember-cli/pull/7141
* Need beta release...
* Migrate away from `ember-cli-qunit` to `ember-qunit` https://github.com/ember-cli/ember-cli/pull/7365

Reply:
Action item is to research and decide on using prettier for CLI development only (not part of blueprints).

Reply:
> Migrate away from ember-cli-qunit to ember-qunit ember-cli/ember-cli#7365

@rwjblue what did you discuss RE ☝️ ?

Reply:
@Turbo87 - As I've mentioned a few times now, I am personally in favor of this. We decided to defer actual discussion until you were able to attend, because I recall you having thoughts on it...